chapters/connectors.tex

```diff
@@ -922,8 +932,11 @@ \subsection{Generation of Connection Graph Equations}\label{generation-of-connec
 \item
   For every optional spanning-tree edge (i.e., a \lstinline!connect(A, B)! equation), in one of the spanning trees, the connection equations are generated according to \cref{generation-of-connection-equations}.
 \item
-  For every optional spanning-tree edge not in any of the spanning trees, the connection equations are generated according to \cref{generation-of-connection-equations}, except for overdetermined type or record instances \lstinline!R!.
+  For every optional non-redundant spanning-tree edge not in any of the spanning trees, the connection equations are generated according to \cref{generation-of-connection-equations}, except for overdetermined type or record instances \lstinline!R!.
   Here the equations \lstinline!0 = R.equalityConstraint(A.R, B.R)! are generated instead of \lstinline!A.R = B.R!.
+\item
+  For every redundant and removed spanning-tree edge (i.e., a \lstinline!connect(A, B)! equation), not in any of the spanning trees, the connection equations for flow-elements are generated according to \cref{generation-of-connection-equations}.
```

Comment:
This section applies only to connections of overconstrained types and according to 9.4.1:
> An overdetermined type or record may neither have flow components nor may be used as a type of flow components.

There should be no separate rule for flow elements. Or is it one of those things where some tools support overconstrained flow variables? Then we also need to update 9.4.1.

Reply:
This made me realize that part of the problem is that the previous text was somewhat confusing. It was discussing connect-equations as though they were breakable branches, even though it is only the overdetermined components in them that generate breakable branches.

I thus first tried to fix that, and then add this correction on top.